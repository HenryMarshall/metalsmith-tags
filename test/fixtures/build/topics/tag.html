<h1>Posts for tag</h1>
<ul>
	<li><small>19/07/2014</small> test</li>

	<li><small>30/06/2014</small> test page 2</li>
</ul>