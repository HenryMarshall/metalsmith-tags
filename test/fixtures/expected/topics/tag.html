<h1>Posts for tag</h1>
<ul>
	<li><small>20/07/2014</small> test</li>

	<li><small>01/07/2014</small> test page 2</li>
</ul>